destination/destination_integration_test.go

```diff
@@ -266,3 +266,84 @@ func assertWrittenRecordsOnStream(
 	recs := testutils.GetRecords(ctx, is, streamName)
 	is.Equal(len(records), len(recs))
 }
+
+func TestWrite_CreateStreamIfNotExists(t *testing.T) {
+	getTestConfig := func(streamName string) map[string]string {
+		cfg := testutils.GetTestConfig(streamName)
+		cfg["createIfNotExists"] = "true"
```

Comment:
since default value is `true`, I would also like to see a test when this value is set to `false` by the user.